lib/network/tsnetwork_bwlimit.c

```diff
@@ -199,9 +199,9 @@ network_bwlimit_get(int op, size_t * len)
 
 	/* Return the appropriate value. */
 	if (op == NETWORK_OP_READ)
-		*len = limit_read.bucket;
+		*len = (size_t)(limit_read.bucket + 0.5);
```

Comment:
Not certain if you wanted to round it, or simply truncate.  It's unlikely that anybody would notice the difference ("I wanted 100,000 bytes per second, not 99,999!  And my network connection is so powerful that I'm sure the difference is due to floating-point representation inexactness, not anything else!"), but I figured that rounding would be safer.